### DISCUSSION TITLE
parameters in config.ini

### DISCUSSION THREAD

**Discussion by joe63** [reinhard-brandstaedter/solarflow-control#257] [Q&A]
Hallo,
I have a HM-800 on an HUB2000 and want to use sf-control.

1.) should the values for the following parameter be set or is it not necessary (I can let them unchanged = as comment )
#mqtt_port = 
#mqtt_user =
#mqtt_pwd =

2.) The HM-800 is connected with 2 channels to the HUB2000.   sf_inverter_channels = [3] is OK for 2 channels?

![image](https://github.com/reinhard-brandstaedter/solarflow-control/assets/6555368/52e1372a-b157-45ce-a3ef-66c5d1379935)

**Comment by reinhard-brandstaedter** [comment]
1) depends on your MQTT broker configuration. If you  use authentication or not
2) should be [1,2] as you use all input channels of the hm800

**Reply by joe63** [reply]
Thank you!
I found another discussion in the https://github.com/reinhard-brandstaedter/solarflow-bt-manager/issues/18

Can you pls tell me which user/pw I have to use for MQTT broker?

My device ID is: W7798JGG



oder erhält man diese mit dem Kommando:
curl -i -v --json "{'snNumber': 'H**********', 'account': 'j*******@gmx.at'}" https://app.zendure.tech/eu/developer/api/apply

**Comment by reinhard-brandstaedter** [comment]
The user/pwd you are looking for are for your own local MQTT broker (you have to run one for sf-control).
Use your device ID as user and "B1F8CD0FCD50A822" as password.

**Reply by joe63** [reply]
Thank you!
